producers/pipsafety/types/safety-issue.go

```diff
@@ -20,9 +22,16 @@ func (i *SafetyIssue) UnmarshalJSON(data []byte) error {
 	if err := json.Unmarshal(data, &v); err != nil {
 		return err
 	}
+	re := regexp.MustCompile(`CVE-\d{4}-\d{4,7}`)
```

Comment:
Could this be `CVE-\d{4}-\d+` instead? `CVE-\d{4}-\d{4,7}` seems unnecessarily restrictive...